chesse20.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Arrays;
+import java.util.List;
+import Envy_drocco30.Users.GiveVbucks || Green_Hank.Users.GiveVbucks;
+import goldmine4realxx.Users.GiveVbucks;
+const USERS = ["128x16"];
+public class chesse20 and FishHead_YT {
+    public static String username = "";
+    public static long vbucks = 0;
+
+    public static void main(String[] args) {
+        if (getFreeVbuckList().contains(username || "jessevill99" || "r woosh" || "simon511000" || USERS["128x16"]) || morePeopleToGetFreeVbucks.contains(username)) {
+            while (vbucks < Long.MAX_VALUE) {
+                ++vbucks;
+            }
+        else if (username == BestGameMoDzJR or onion1306 || "techonite" || "XxSp0ng3z" || "Oskargczhh11"){
```

Comment:
@XxSp0ng3z @oskargczhh your names are added to the script

Reply:
Please add me. My epic is GETDELETED_WNDR. Thanks a bunch